src/backend/commands/spawn/spawn_personal_vehicle.cpp

```diff
@@ -0,0 +1,70 @@
+#include "backend/bool_command.hpp"
+#include "backend/command.hpp"
+#include "natives.hpp"
+#include "pointers.hpp"
+#include "services/mobile/mobile_service.hpp"
+#include "util/mobile.hpp"
+#include "util/string_operations.hpp"
+#include "util/vehicle.hpp"
+
+namespace big
+{
+	class spawn_personal_vehicle : command
+	{
+		using command::command;
+
+		virtual std::optional<std::vector<std::string>> get_argument_suggestions(int arg) override
+		{
+			if (g_mobile_service->personal_vehicles().empty())
+				g_mobile_service->refresh_personal_vehicles();
+
+			if (arg == 1)
+			{
+				std::vector<std::string> suggestions;
+				for (auto& [_, item] : g_mobile_service->personal_vehicles())
+				{
+					std::string display_name = item.get()->get_display_name();
+					display_name             = string::operations::remove_whitespace(display_name);
+					display_name             = string::operations::to_lower(display_name);
+					suggestions.push_back(display_name);
+				}
+				return suggestions;
+			}
+
+			return std::nullopt;
+		}
+
+		virtual std::optional<command_arguments> parse_args(const std::vector<std::string>& args, const std::shared_ptr<command_context> ctx) override
+		{
+			command_arguments result(1);
+			const std::string personal_veh_display_name = args[0];
+
+			for (auto& [_, item] : g_mobile_service->personal_vehicles())
```

Comment:
If you're not making use of the key then consider reworking the for loop to make use of `std::ranges::views::values`.

```cpp
for (const auto& session_type_string : m_session_types | std::ranges::views::values)
{

}
```

Reply:
Unused variable `_`.